tests/block_tools.py

```diff
@@ -257,25 +287,33 @@ def change_config(self, new_config: Dict):
             save_config(self.root_path, "config.yaml", self._config)
 
     def add_plot_directory(self, path: Path) -> None:
-        self._config = add_plot_directory(self.root_path, str(path))
+        try:
+            self._config = add_plot_directory(self.root_path, str(path))
```

Comment:
Why do we need to catch the value error here?

Reply:
It will throw an error if the plot directory is already in the config, let me see if i can solve that better